ppr-ui/src/store/getters/state-getters.ts

```diff
@@ -610,3 +610,23 @@ export const hasMorePages = (state: StateIF): boolean => {
 export const getMhrHomeSections = (state: StateIF): Array<HomeSectionIF> => {
   return state.stateModel.mhrRegistration.description.sections
 }
+
+export const getMhrRegistrationManufacturerName = (state: StateIF): string => {
+  return state.stateModel.mhrRegistration.description.manufacturer
+}
+
+export const getMhrRegistrationYearOfManufacture = (state: StateIF): number => {
+  return state.stateModel.mhrRegistration.description.baseInformation.year
+}
+
+export const getMhrRegistrationHomeMake = (state: StateIF): string => {
+  return state.stateModel.mhrRegistration.description.baseInformation.make
+}
+
+export const getMhrRegistrationHomeModel = (state: StateIF): string => {
+  return state.stateModel.mhrRegistration.description.baseInformation.model
+}
+
+export const getMhrRegistrationOtherInfo = (state: StateIF): string => {
+  return state.stateModel.mhrRegistration.description.otherRemarks
```

Comment:
I didn't see `otherRemarks` or `other` in the spec, is this something Doug is still working on perhaps?